api/src/main/java/javax/enterprise/concurrent/ManagedScheduledExecutorService.java

```diff
@@ -23,20 +23,20 @@
  *
  * A ManagedScheduledExecutorService extends the Java&trade; SE ScheduledExecutorService
  * to provide methods for submitting delayed or periodic tasks for execution in
- * a Java&trade; EE environment.
+ * a Jakarta&trade; EE environment.
  * Implementations of the ManagedScheduledExecutorService are
- * provided by a Java&trade; EE Product Provider.  Application Component Providers
+ * provided by a Jakarta&trade; EE Product Provider.  Application Component Providers
  * use the Java Naming and Directory Interface&trade; (JNDI) to look-up instances of one
  * or more ManagedScheduledExecutorService objects using resource environment references.
  * ManagedScheduledExecutorService instances can also be injected into application
  * components through the use of the {@code Resource} annotation.<p>
  *
- * The Concurrency Utilities for Java&trade; EE specification describes several
+ * The Jakarta Concurrency specification describes several
```

Comment:
This is pretty minor, but in two places (such as above), `Concurrency Utilities for Java&trade; EE specification` was changed to `Jakarta Concurrency specification`, whereas elsewhere it was changed to `Jakarta EE Concurrency specification`. We should make all of these the same for consistency.

Reply:
good catch it should be `Jakarta Concurrency`